releases/nightly-build/compose-files/docker-compose-nexus-arm64.yml

```diff
@@ -27,8 +27,10 @@ services:
   app-service-rules:
     container_name: edgex-app-service-configurable-rules
     depends_on:
-    - consul
-    - data
+      consul:
+        condition: service_started
+      data:
+        condition: service_started
```

Comment:
Please down grade your docker-compose to version 1.27.2.

Latest version has a bug that is generating the `depends_on` section incorrectly for the 3.x spec and this breaks the TAF tests.

Reply:
Done